app-interface/app_sre_cronjob_runner_build_push.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+IMAGE_NAME="quay.io/app-sre/rhose-cronjob-runner"
+# the RHOSE CronJob Runner is stable, we do not need to push a new image for every commit
+IMAGE_TAG="1.0"
+
+docker build -f app-interface/cronjobs/Dockerfile.cronjobs -t "${IMAGE_NAME}:${IMAGE_TAG}"  .
```

Comment:
Maybe it would be better to move the Dockerfile.cronjobs to `/docker` folder, WDYT?

Reply:
I think it is ok to keep it here as the image is used just internally.